ctk/common/src/main/kotlin/org/codice/compliance/verification/core/SamlDefinedIdentifiersVerifier.kt

```diff
@@ -124,6 +134,21 @@ internal class SamlDefinedIdentifiersVerifier(val node: Node) {
         }
     }
 
+    /** 8.3 Name Identifier Format Identifiers */
+    private fun verifyNameIdentifierFormatIdentifiers() {
+        node.recursiveChildren()
+                .filter { it.attributeText("Format") == EMAIL_URI }
+                .forEach {
```

Comment:
I didn't use `.any` here because I wanted to use the actual Identifier in the error message.